src/deploy/deployer_easyAuction.ts

```diff
@@ -19,6 +22,24 @@ const deployEasyContract: DeployFunction = async function (
     log: true,
     deterministicDeployment: false,
   });
+  const easyAuctionDeployed = await get(easyAuction);
+  let weth9Address;
+  const chainId = (await hre.ethers.provider.getNetwork()).chainId;
+  if (chainId == 4) {
+    weth9Address = weth9Networks.WETH9["4"]["address"];
+  } else if (chainId == 1) {
+    weth9Address = weth9Networks.WETH9["1"]["address"];
+  } else if (chainId == 100) {
+    weth9Address = "0xe91D153E0b41518A2Ce8Dd3D7944Fa863463a97d";
+  }
```

Comment:
The hardhat case is not considered, running `yarn deploy` throws an error.
In the test case you should also deploy a new `WETH` instance.

Reply:
hardhat tests don't rely on the deployment process, right?

Or arre you saying that we should reuse thee logic of the migration within the tests?

Reply:
It would be nice to have extra tests for the deployment process itself.
In any case, I always use `yarn deploy` to test if the deployment scripts are running correctly, while now I'd have to choose a network and deploy there.

Reply:
I see... created an issue: https://github.com/gnosis/ido-contracts/issues/86